R/remove_brackets.R

```diff
@@ -0,0 +1,43 @@
+#' Remove content between brackets
+#' 
+#' @param string A character vector to clean
+#' @param bracket Type of bracket containing the content to remove.
+#'   Must be one of "(" (default), "<", "["
+#' 
+#' @returns A character vector of the same size of `string`, where
+#'   elements has been stripped of their `bracket`ted content.
+#' 
+#' @export
+#' 
+#' @examples
+#' remove_brackets(
+#'   "R source code is versioned with svn (and mirrored on GitHub)"
+#' )
+#' 
+#' remove_brackets(
+#'   c("r-project.org <https://r-project.org>", "python.org <https://python.org>"),
+#'   bracket = "<"
+#' )
+#' 
+#' remove_brackets(
+#'   "Content in square bracket [is removed] in the middle of a string",
+#'   bracket = "["
+#' )
+remove_brackets <- function(string, bracket = c("(", "<", "[")) {
+  
+  bracket <- match.arg(bracket)
+
+  closing_bracket <- switch(
+    bracket,
+    "<" = ">",
+    "(" = ")",
+    "[" = "]"
+  )
+  
+  string |> 
+    stringi::stri_replace_all_regex(
```

Comment:
Trying to use this creates rather opaque warnings that may seem ominous to new users. 

```
Warning message:
In stringi::stri_replace_all_regex(string, paste0("\\", bracket,  :
  argument is not an atomic vector; coercing
```

Can we clarify these?